plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBReportMediatorPropertiesEditionComponent.java

```diff
@@ -655,7 +674,8 @@ public void updatePart(Notification msg) {
 			EsbPackage.eINSTANCE.getAbstractSqlExecutorMediator_PropertyValidationquery(),
 			EsbPackage.eINSTANCE.getAbstractSqlExecutorMediator_PropertyInitialsize(),
 			EsbPackage.eINSTANCE.getAbstractSqlExecutorMediator_SqlStatements(),
-			EsbPackage.eINSTANCE.getDBReportMediator_ConnectionUseTransaction()		);
+			EsbPackage.eINSTANCE.getDBReportMediator_ConnectionUseTransaction(),
+			EsbPackage.eINSTANCE.getMediator_Reverse()		);
```

Comment:
Why did we change this?

Reply:
Due to code generation the code line had been added